### PR TITLE
Support Regional Content

### PR DESCRIPTION
**Pull request by VenierGiacomo**
## Description
Create a `Component Picker` that customizes pages based on the IP (`region`) of the visitor 

Remove Testimonials from contentful as we had reached the limit of components

## Test
Visit https://deploy-preview-688--ledgy.netlify.app/test-landingpage/  and see how the contnet of the page changes, based on the region. (Specifically, logos and the features grid)
Eg.
[UK](https://deploy-preview-688--ledgy.netlify.app/uk/test-landingpage/)
[DE](https://deploy-preview-688--ledgy.netlify.app/de/test-landingpage/)

image
image

### CODE REVIEW

File: src/@types/contentful.d.ts [hunk 1]
```diff
@@ -264,3 +265,16 @@ declare type CompetitorTableProps = Id & {
   competitorName: string;
   tableSection: CompetitorTableSectionProps[];
 };
+
+declare type RegionalComponentPickerEntriesProps = Id & {
+  __typename: 'ContentfulRegionalComponentPickerEntries';
+  global: EntryProps;
+  uk: EntryProps;
+  de: EntryProps;
```
Comment (thorhildurt) [inline src/@types/contentful.d.ts:273]: When testing for the DE, do we want to display this text in English?
Screenshot 2022-06-10 at 12 35 28

File: src/components/EntryPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { ComponentPicker } from './ComponentPicker';
+import { RegionalComponentPicker } from './RegionalComponentPicker';
+
+export const EntryPicker = ({
+  entries,
+  ...baseProps
+}: {
+  prefix: string;
+  region: 'global' | 'uk' | 'de' | 'fr';
```
Comment (thorhildurt) [inline src/components/EntryPicker.tsx:10]: We can have a specific type for the region string. Something like: 
`const REGION_TYPE = 'global' | 'uk' | 'de' | 'fr';`
That would be nice if we need to re-use it or when we need to add more region values compared to doing that individually when defining function properties.

**Review by thorhildurt — COMMENTED** [review]
Looks good, added two comments 💬

File: src/components/EntryPicker.tsx (hunk 2 above)
Comment (VenierGiacomo) [reply src/components/EntryPicker.tsx]: Yes, you are right! 
I had forgotten to push

Comment (VenierGiacomo) [reply src/components/EntryPicker.tsx]: We already had both the region and the Prefix type

File: src/@types/contentful.d.ts (hunk 1 above)
Comment (VenierGiacomo) [reply src/@types/contentful.d.ts:273]: In theory yes, but if the copy has not been translated yet, we serve it in english

**Review by thorhildurt — APPROVED** [review]
Tested the preview branch for all available regions and it works well 🚀

**Pull request merged**